Completes rough version of start page.

Jack-Gill opened the pull request:
Adds debug nav to current 'home' page, so you can get to the start page.
'Ready' button links from start page to home page, placeholder until fsm implemented.
Changes 'Genuis' to 'Songuage' in a few places - RIP.

Jack-Gill commented:
I did forget we were using TypeScript :P Will rewrite in my next commit!

ChristienGuy commented on src/pages/Start.styles.js:
```diff
@@ -0,0 +1,43 @@
+`;
+
+const ReadyButton = styled.button`
+background-color: ${({theme}) => theme.COLOURS.accent};
```
We should get some autoformatting stuff running somewhere, at some point. Probably not important at all right now tho haha.

Jack-Gill replied:
Definitely! I actually had to add prettier to this PR because otherwise 'toMatchInlineSnapshot' threw an error (https://github.com/facebook/jest/issues/8467)...

ChristienGuy commented on src/pages/Home.js:
```diff
@@ -111,6 +112,16 @@ class Home extends Component {
+                        backgroundColor: 'white',
+                    }}
+                >
+                    <RouterLink to="/start">Start Screen</RouterLink>
```
I wouldn't worry too much about wiring up all these URLs, we're not going to use them for managing the game or anything

Jack-Gill replied:
Yeah, it's just a stopgap until we've got the FSM managing this rendering - just makes it slightly easier to get to me start page so you can look at it! I assume this Home page will be completely re-written by the time we get to MVP.

Jack-Gill commented:
This is ready for you to take a quick look @ChristienGuy - I moved 'Start' to components and removed the test because I couldn't work out how to test it now that I'm passing it the gameService (Admittedly, I did not try very hard!)
